content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -190,40 +190,40 @@ Rechenschaftspflicht ist wichtig, deshalb wollen wir dieses Jahr mit der Veröff
 
 ### Strafverfolgungsbehörden
 
 Bitte lesen Sie unsere Leitlinien hier: [https://matrix.org/legal/law-enforcement-guidelines/](https://matrix.org/legal/law-enforcement-guidelines/)
 
 ### Finanzierung
 
-Die Tech-Industrie investiert zu wenig in die Sicherheit. Wir versuchen, es besser zu machen, und Sicherheit ist der größte Posten im Haushalt der Stiftung. Diese Investition erfolgt trotz der Herausforderungen, denen wir uns bei unseren [ständigen](https://matrix.org/blog/2024/12/25/the-matrix-holiday-special-2024/) Versuchen [gegenübersehen](https://matrix.org/blog/2024/01/2024-roadmap-and-fundraiser/), [Mittel zur Unterstützung](https://matrix.org/blog/2024/01/2024-roadmap-and-fundraiser/) der Entwicklung von Matrix [und](https://matrix.org/blog/2024/04/open-source-publicly-funded-service/) [offener Software im Allgemeinen](https://matrix.org/blog/2024/07/17/ngi-open-letter/) aufzubringen. Wir sind für unseren Betrieb auf Spenden angewiesen. Große öffentliche und private Organisationen nutzen die von uns geleistete Arbeit, oft ohne einen finanziellen Beitrag zu leisten. Es wäre einfach, die Ausgaben für Vertrauen und Sicherheit angesichts dieser wirtschaftlichen Zwänge zu opfern, aber wir versuchen, einen besseren Weg zu finden. Wenn Sie unsere Arbeit im Bereich Sicherheit finanziell unterstützen möchten, wenden Sie sich bitte an die Stiftung unter [funding@matrix.org](mailto:funding@matrix.org).
\ No newline at end of file
+Die Tech-Industrie investiert zu wenig in Schutz. Wir versuchen, es besser zu machen, und Schutz ist der größte Posten im Haushalt der Matrix.org Stiftung. Diese Investition erfolgt trotz der Herausforderungen, denen wir uns bei unseren [ständigen](https://matrix.org/blog/2024/12/25/the-matrix-holiday-special-2024/) Versuchen [gegenübersehen](https://matrix.org/blog/2024/01/2024-roadmap-and-fundraiser/), [Mittel zur Unterstützung](https://matrix.org/blog/2024/01/2024-roadmap-and-fundraiser/) der Entwicklung von Matrix [und](https://matrix.org/blog/2024/04/open-source-publicly-funded-service/) [Open Source Software im Allgemeinen](https://matrix.org/blog/2024/07/17/ngi-open-letter/) aufzubringen. Wir sind für unseren Betrieb auf Spenden angewiesen. Große öffentliche und private Organisationen nutzen die von uns geleistete Arbeit, oft ohne einen finanziellen Beitrag zu leisten. Es wäre einfach, die Ausgaben für Trust & Safety angesichts dieser wirtschaftlichen Zwänge zu opfern, aber wir versuchen, einen besseren Weg zu finden. Wenn Sie unsere Arbeit im Bereich Schutz finanziell unterstützen möchten, wenden Sie sich bitte an die Stiftung unter [funding@matrix.org](mailto:funding@matrix.org).
```

Comment:
Github doing a horrible job at this diff:

This is just a change with the newline right?